app/controllers/insured/family_members_controller.rb

```diff
@@ -12,7 +12,15 @@ def index
     if (params[:resident_role_id].present? && params[:resident_role_id])
       @type = "resident"
       @resident_role = ResidentRole.find(params[:resident_role_id])
-      @family.hire_broker_agency(current_user.person.broker_role.try(:id))
+      begin
+        @family.hire_broker_agency(current_user.person.broker_role.try(:id))
+      rescue StandardError => e
+        exception_message = "Error: #{e}"
+        exception_message += "Unable to find family for person #{@person&.hbx_id}." if @family.blank?
+        Rails.logger.error(exception_message) unless Rails.env.test?
+        puts(exception_message) unless Rails.env.test?
```

Comment:
@ZASMan  do we need both Rails.logger and puts statements here?

Reply:
nah ill get rid of the puts statement